api/nakadi-event-bus-api.yaml

```diff
@@ -1503,11 +1535,37 @@ definitions:
         type: string
         default: 'random'
 
+      compatibility_mode:
+        description: |
+          Compatibility modes provide a mean for event owners to evolve their schema, given changes
+          respect the semantics defined by this field.
+
+          It's designed to be flexible enough so that producers can evolve their schemas while not
+          inadvertently breaking existent consumers.
+
+          Once defined, the compatibility mode is fixed, since otherwise it would break a predefined contract,
+          jeopardizing the value of this feature.
+
+          List of available compatibility modes:
+
+          - 'deprecated': Schema changes are not allowed. This is used only by legacy event types. This option is not
```

Comment:
Besides been "fixed" in the sense that schema changes are not allowed, validation of events sent to these topics is "lenient". By default, schema validation allows additional properties even when not explicitly stated. It's not just about schema evolution but also about important changes to the way events are going to be validated from now on.

Also, it might include some feature that are no longer accepts, such as "not" attributes and "patternProperties".

Our goal is to move these events from deprecated to the other possible values. We think that allowing lenient schema validation is harmful and should be avoided.

Having something called "fixed" would indicate that it's a viable option, when it's not.